libraries/AP_BattMonitor/AP_BattMonitor_config.h

```diff
@@ -133,3 +133,8 @@
 #ifndef AP_BATTERY_OPTIONS_PARAM_ENABLED
 #define AP_BATTERY_OPTIONS_PARAM_ENABLED (!defined(HAL_BUILD_AP_PERIPH) || AP_BATTERY_SUM_ENABLED)
 #endif
+
+
+#ifndef AP_BATTERY_JBDCAN_ENABLED
+#define AP_BATTERY_JBDCAN_ENABLED 1
```

Comment:
```suggestion
#define AP_BATTERY_JBDCAN_ENABLED 0
```

You also need to add an entry into `build_options.py` to make this available as an option on the custom build server.